extra/docker/base/Dockerfile

```
@@ -3,26 +3,23 @@
 # Based on Ubuntu
 ############################################################
 
-FROM ubuntu:bionic
+FROM ubuntu:focal
```

We can go even further now.

```suggestion
FROM ubuntu:jammy
```